backend/blogchain/posts/bc/contracts.py

```diff
@@ -204,3 +204,50 @@ def verify_comments(self, comments: Iterable[Comment]):
                     verified += 1
                     break
         return verified
+
+
+class VotesContract:
+
```

Comment:
All these classes probably could be refactored to use some common logic. I will do it at some point